settings.py

```diff
@@ -54,3 +54,6 @@
 LANGUAGES = [
     ('en-gb', 'English'),
 ]
+
+PAGINATE_EVENTS_BY = 10
+DEFAULT_TIME_DIRECTION = ""  # chose between "", "upcoming" & "previous" - "" gets all published events
```

Comment:
We need to make this more robust and check they've entered a correct string here